config/routes.rb

```diff
@@ -348,7 +348,7 @@
       get '/in_person' => 'in_person#index'
       get '/in_person/ready_to_verify' => 'in_person/ready_to_verify#show',
           as: :in_person_ready_to_verify
-      get '/in_person/usps_locations' => 'in_person/usps_locations#index'
+      post '/in_person/usps_locations' => 'in_person/usps_locations#index'
```

Comment:
if this was actively used in production, I'd recommend allowing for both verbs temporarily (because our deploys take ~15 minutes) so requests can go from the frontend to a host running old code potentially

Reply:
We deploy the frontend together with the backend, but the issue is that sometimes old boxes are up. So if you happen to load a page from a new box, it'll load with new JS. But when that JS makes a request, it'll go through the load balance to one of any number of hosts, and for ~15minutes that could be an old one too